services/github/project/api/routes.py

```diff
@@ -200,7 +210,30 @@ def get_items():
                     item['reviewer_points'] = 0
                 item['num_reactions'] = r.get('reactions').get('totalCount') or 0
                 item['title'] = r.get('title')
-                items.append(item)
+                # check if date is between start and end date
+                if start_creation_date != None and end_creation_date != None:
```

Comment:
Please create tests here that ensure we are calculating the data math correctly [here](https://github.com/sendgrid/dx-automator/blob/development/services/github/project/tests/test_github.py) and that we are properly throwing exceptions when the data formats given are not correct.